sw/device/lib/dif/autogen/dif_pinmux_autogen_unittest.cc

```diff
@@ -0,0 +1,27 @@
+// Copyright lowRISC contributors.
```

Comment:
nit: this file doesn't need to be generated as well.

Reply:
same as above